src/components/AddBook.js

```diff
@@ -16,6 +16,21 @@ const Form = () => {
     setAuthor(e.target.value);
   };
 
+  const bookToAPI = async (book) => {
+    await fetch('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/YM3PSnyKoYXp8BbKCQ5a/books', {
+      method: 'POST',
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        title: `${book.name}&&&${book.author}`,
+        category: `${book.category}`,
+        item_id: `${book.id}`,
```

Comment:
- [ ] The API should be in the redux to store the data and push it to the `action.types`, now you are using it like the JS module creating a **Json.stringify** to add it to the body.

![ooo](https://user-images.githubusercontent.com/61562626/140185509-bda7875a-0e1f-4405-b407-0bc82e99f2c4.jpeg)

Reply:
Done, thanks for the review.